content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -355,6 +355,19 @@ Used on: PersistentVolumeClaim
 
 This annotation will be added to dynamic provisioning required PVC.
 
+### volumes.kubernetes.io/controller-managed-attach-detach
+
+Used on: Node
+
+If a node has set the annotation `volumes.kubernetes.io/controller-managed-attach-detach`
+on itself, then its storage attach and detach operations are being managed
+by the controller running on the cluster’s master.
+The cluster’s master controller automatically inspects all nodes for this annotation
```

Comment:
> The cluster’s master controller automatically inspects all nodes for this annotation

It's not the “master controller”. @Kartik494 would you be willing to ask SIG Storage folks which controller it actually is?

Reply:
Some candidates: https://github.com/kubernetes/kubernetes/tree/master/pkg/controller/volume

Reply:
Sure i will do this

> > The cluster’s master controller automatically inspects all nodes for this annotation
> 
> It's not the “master controller”. @Kartik494 would you be willing to ask SIG Storage folks which controller it actually is?

Reply:
@kubernetes/sig-storage-pr-reviews i am a little confuse on opting for which controller inspect all nodes for this annotation. Could you please help me out.

Reply:
@sftim can you ping sig-storage pr reviewers to take a look into this,

Reply:
@Kartik494 would **_you_** be willing to ask SIG Storage folks which controller it actually is?

Reply:
It's attach/detach controller running in kube-controller-manager who inspects and acts on the annotation, for example [here](https://github.com/kubernetes/kubernetes/blob/6bdda2da160043c5fcdfd25bbea9e1c1b804c9e5/pkg/controller/volume/attachdetach/attach_detach_controller.go#L899)

And node annotates itself based on `kubelet --enable-controller-attach-detach` cmdline flag or via corresponding kubelet config file field.

Reply:
@jsafrane thanks for suggestion on this , @sftim could you please take a look and let me know if this sounds good to you.

Reply:
Thanks @jsafrane